### ISSUE TITLE
enhancement request - add reminder ahead of expiry 

### ISSUE THREAD

**Issue by snigehere** [jampez77/DVLA-Vehicle-Enquiry-Service#19]
Would it be possible to add a calendar entry xx days (where xx is either fixed or able to be configured) ahead of expiry of MOT and tax to allow time for it to be done ahead of expiry? 

I could do an automation myself but it would be tidier if it was in the one place 

Calendar entry title like  - MOT due - <reg> - <dd-mmm-yy> or TAX due - <reg> - <dd-mmm-yy>

Many thanks

**Comment by jampez77** [comment]
Sorry man, I probably won't be doing this as, like you say it can be done in an automation.

**Comment by snigehere** [comment]
No worries - thanks for considering it ... time for me to learn about calendars ...

**Comment by jampez77** [comment]
I've found automating from the calendars to be quite tricky, especially when a calendar have multiple events at the same time. 

I found it easier using the `sensor.dvla_<reg_number>_motexpirydate` and `sensor.dvla_<reg_number>_taxduedate` entities.

Hope this helps, also, thanks a lot for your bugs/issues. You've helped make this integration stronger and I very much appreciate that!

If you have any upcoming flights or incoming parcels then feel free to check out my other integrations :D
